components/engine_rocks/src/logger.rs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+use rocksdb::{DBInfoLogLevel as InfoLogLevel, Logger};
+
+#[derive(Default)]
```

Comment:
add a comment for me? we need to add `Logger` interface to `engine_trait` crate, but let's postpone for later.
```
// TODO(yiwu): abstract the Logger interface.
```